blockonomics-woocommerce.php

```diff
@@ -243,121 +240,228 @@ function show_basic() {
-                                    <option value="1" <?php selected(get_option('blockonomics_network_confirmation'), 1); ?>>1</option>
-                                    <option value="0" <?php selected(get_option('blockonomics_network_confirmation'), 0); ?>>0</option>
-                                </select></td>
-                        </tr>
-                    </table>
-                </div>
-                <p class="submit">
-                    <input type="submit" class="button-primary" value="Save"/>
-                    <input type="hidden" name="action" value="update" />
-                    <input type="hidden" name="page_options" value="blockonomics_api_key,blockonomics_bch,blockonomics_timeperiod,blockonomics_margin,blockonomics_gen_callback,blockonomics_api_updated,blockonomics_underpayment_slack,blockonomics_lite,blockonomics_nojs,blockonomics_network_confirmation" />
-                    <input onclick="checkForAPIKeyChange();" class="button-primary" name="test-setup-submit" value="Test Setup" style="max-width:85px;">
-                </p>
-            </form>
-            <form method="POST" name="testSetupForm">
-                <p class="submit">
-                    <input type="hidden" name="page" value="blockonomics_options">
-                    <input type="hidden" name="runTest" value="true">
-                </p>
-            </form>
-            <form method="POST" name="generateSecretForm">
-                <p class="submit">
-                    <input type="hidden" name="generateSecret" value="true">
-                </p>
-            </form>
-    </div>
+                        </table>
+
+                    <h1 class="bnomics-options-small-intendation">
+                        <input type="checkbox" name="blockonomics_bch" value="1"<?php checked("1", get_option('blockonomics_bch')); ?>" />
+                        Bitcoin Cash (BCH)
+                    </h1>
+                    <p class="bnomics-options-medium-intendation">To configure, click <b> Get Started for Free </b> on
+                        <a href="https://bch.blockonomics.co.merchants">https://bch.blockonomics.co.merchants</a>
+                    </p>
+                        <table class="form-table bnomics-options-intendation-heading">
+                        <?php 
+                        $bch_enabled = get_option("blockonomics_bch");
+                        if ($bch_enabled):  ?>
+                        <th scope="row"><h1>Destination</h1></th>
+                                <td colspan="2" class="bnomics-options-no-padding">
+                                    <?php
+                                    $total_received = get_option('blockonomics_temp_withdraw_amount') / 1.0e8;
+                                    $api_key = get_option("blockonomics_api_key");
+                                    $temp_api_key = get_option("blockonomics_temp_api_key");
```

Comment:
Temp wallet functionality can be removed from BCH.

Reply:
True, that also makes the code more concise and easier to work with. I've made the changes and pushed them.